lib/mocha.js

```diff
@@ -290,32 +283,6 @@ Mocha.prototype.loadFiles = function(fn) {
   fn && fn();
 };
 
-/**
- * Implements desktop notifications using a pseudo-reporter.
- *
- * @private
- * @see {@link Mocha#growl}
- * @param {Object} runner - Runner instance.
- * @param {Object} reporter - Reporter instance.
- */
-Mocha.prototype._growl = function(runner, reporter) {
-  var notify = require('growl');
-
-  runner.on('end', function() {
-    var stats = reporter.stats;
-    if (stats.failures) {
-      var msg = stats.failures + ' of ' + runner.total + ' tests failed';
-      notify(msg, {name: 'mocha', title: 'Failed', image: image('error')});
-    } else {
-      notify(stats.passes + ' tests passed in ' + stats.duration + 'ms', {
-        name: 'mocha',
-        title: 'Passed',
-        image: image('ok')
```

Comment:
Was left wondering why this wasn't triggering browser test errors. Via `browserify`, the `require('growl')` was translated to a `noop()` and `path` module was set to `false`. Yet in order to invoke `notify` (a.k.a. `noop`), its arguments would need to be assembled first. When creating the options, the `image` function should have blown up attempting to `path.join`, no?

Reply:
gonna guess it's because `growl: true` was never set, which means this function would never get called.